G/Geant4/build_tarballs.jl

```diff
@@ -101,4 +109,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"8", julia_compat="1.6")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"11", julia_compat="1.6")
```

Comment:
GCC 11 can be problematic for Julia v1.6, for C++ code: https://github.com/JuliaPackaging/Yggdrasil/blob/9e93401baf07c15c00d239fc6c4d124f8121451f/CONTRIBUTING.md#compatibility-tips.  Would GCC 10 be sufficient?

Reply:
Should be also OK I guess. I'll run a test

Reply:
Done. The release notes says that has also been tested with gcc10